docs/guideline.md

```diff
@@ -0,0 +1,111 @@
+
+##### Toolings
+
+* [Horizon Bridge Subgraphs](https://github.com/harmony-one/bounties/issues/15)
+* [Harmony Subgraphs: Transactions, HRC20 and HRC721](https://github.com/harmony-one/bounties/issues/10)
+* [analytics.harmony.one](https://github.com/harmony-one/bounties/issues/39)
+
+#### 1.2.2 External bounties
+
+##### External collaborations and integrations
+
+* Bridges ([Terra](https://github.com/harmony-one/bounties/issues/32), [Celo](https://github.com/harmony-one/bounties/issues/33))
+* API integration ([0x](https://github.com/harmony-one/bounties/issues/38), [Trezor](https://github.com/harmony-one/bounties/issues/29), [UMA](https://github.com/harmony-one/bounties/issues/12))
+* Misc ([Humming bot](https://github.com/harmony-one/bounties/issues/16))
+
+##### Ecosystem and DAPPS
+
+* [Argent Wallet](https://github.com/harmony-one/bounties/issues/31)
+* [Harmony Name Service](https://github.com/harmony-one/bounties/issues/13)
+* [PoolTogether](https://github.com/harmony-one/bounties/issues/44)
+* [Indonesia Rupiah Token (IDRT) stablecoin](https://github.com/harmony-one/bounties/issues/11)
+
+#### 1.2.3 Research bounties
+
+* [1s Finality](https://github.com/harmony-one/bounties/issues/14)
+* [nnet](https://github.com/harmony-one/bounties/issues/25)
+* [A lot more ideas](https://docs.google.com/spreadsheets/u/1/d/1J3gMrq52Ho75fpw716Am_5akjVsg6U5brHHSy1z4tDM/edit#gid=527101002)
+
+## 2 "Good" bounties
+
+### 2.1 What makes a good bounty?
+
+First, what is the characteristics of bounties?
+
+1. The candidates are all developers. And most of them work along.
+2. The candidate may not have prior knowledge to Harmony blockchain as well as the context.
+3. The candidate does not expect to run in long-term. They walk in and go.
+
+This will give several insights to what a good bounty should be:
+
+1. Self-contained.
+
+   This means the bounty would better not require help or support from anyone or anything else during the process, it is complete by itself. 
+
+   If the self-contained cannot be contained for a bounty, you need to provide with a clean scope with lesser boundary.
+
+   Sometimes this can be compromised through clean writing and some helpful conversation at the bootstrap. 
+
+2. No ambiguity in project description.
+
+   For engineers, less ambiguity the better. Treat them as high-level coding machine. You need to strictly define the starting point and target point, and let them improvise in the process. It would be very helpful to define the starting/ending with some engineering documentation.
+
+3. Well defined acceptance criteria.
+
+   This is the "ending point" which we talked about. Do not be afraid to add strict and detailed criterias. Actually more strict you are in acceptance criteria, the happerer the engineer is. Providing some test scenarios would help for both developer and the proposer of the bounty.
+
+4. No business included or long-term dev-ops.
```

Comment:
this is again negotiable, for long-term DevOps, there could be deployment to static website or automation included like terraform/ansible to make it sustainable.